Deletion of Model Versions (#463)
zigbee-alliance/distributed-compliance-ledger: docs/transactions.md

Deletes an existing Model identified by a unique combination of `vid` (vendor ID) and `pid` (product ID)
by the vendor account.

> If one of Model Versions associated with the Model is certified than Model can not be deleted. If Model has no associated certified Model Versions, Model Versions will be removed with associated Model.

Reviewer: ```suggestion
If one of Model Versions associated with the Model is certified then Model can not be deleted. When Model is deleted, all associated Model Versions will be deleted as well.
```

Author: Committed suggestion

- Parameters:

>   - vid: `int32` -  model vendor ID (positive non-zero)

Author: Fixed it

- CLI command:
  - `dcld tx model update-model-version --vid=<uint16> --pid=<uint16> --softwareVersion=<uint32> ... --from=<account>`

#### DELETE_MODEL_VERSION

**Status: Implemented**

Deletes an existing Model Version identified by a unique combination of `vid` (vendor ID), `pid` (product ID) and `softwareVersion`
by the vendor account.

Model Version can be deleted only before it is certified.

- Parameters:

>   - vid: `int32` -  model version vendor ID (positive non-zero)

Reviewer: It's uint16, not int32.
